src/CMakeLists.txt

```diff
@@ -4,12 +4,14 @@ project(SopraCommon)
 set(CMAKE_CXX_STANDARD 17)
 
 set(SOURCE
-	library.cpp
-	)
+		library.cpp
+		Gadget.cpp
+		gadgets/WiretapWithEarplugs.cpp
+		gadgets/Cocktail.cpp
+		util/UUID.cpp
+		)
 
-set(INCLUDE
-	library.h
-	)
+include_directories(.)
```

Comment:
Dann muss aber auch `install(FILES ${INCLUDE} DESTINATION /usr/include/${PROJECT_NAME})` angepasst werden, oder?

Reply:
Aha, habe nicht gesehen dass das für das install target von relevanz ist, habe das noch nie verwendet, werde mich da mal ein bisschen einlesen und das dann anpassen!

Reply:
Also ich habe das mal so hinzugefügt wie du das gemacht hattest, aber das install target geht bei mir immer noch nicht. Muss man mal fixen, ist aber nicht Teil dieses PRs. Das ist für uns wohl sowieso nicht soooo relevant (aber natürlich nice to have), da wir diese lib ja AFAIK als submodul verwenden wollen

Reply:
#7